checkov/openapi/checks/resource/Version.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, Any
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.json_doc.enums import BlockType
+from checkov.openapi.checks.base_openapi_check import BaseOpenapiCheck
+
+
+class Version(BaseOpenapiCheck):
```

Comment:
why a k8 check and not a real check?

Reply:
its a dummy check.

Reply:
please use a real one so it will have context for future maintenance or use "foo/goo" and not k8.